packages/next-plugin-sentry/src/on-error-client.js

```diff
@@ -0,0 +1,5 @@
+import * as Sentry from '@sentry/browser'
+
+export default async function onErrorClient ({ err }) {
+  Sentry.captureException(err)
```

Comment:
Just so I'm understanding the RFC correctly - there would be an escape hatch if we needed to override this while still using the plugin, correct?

Reply:
Not currently.